.vscode/tasks.json

```diff
@@ -157,7 +157,7 @@
       "id": "packagesToIgnore",
       "type": "promptString",
       "description": "Packages To Ignore: These are packages that are not to be included in the change log (Ex package1,package2)",
-      "default": "salesforcedx-vscode-soql"
+      "default": ""
```

Comment:
@jeffb-sfdc this is the change to get rid of that soql default too

Reply:
nice